tests/queries/0_stateless/01293_show_clusters.reference

```diff
@@ -1,2 +1,3 @@
 test_shard_localhost
-test_shard_localhost	1	1	1	localhost	::1	9000	1	default	
+test_cluster_one_shard_two_replicas	1	1	1	127.0.0.1	127.0.0.1	9000	1	default	
```

Comment:
unrelated?

Reply:
Very related to `--network=host` in fasttest python script